etl/steps/data/grapher/ggdc/2022-11-28/penn_world_table.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+from owid import catalog
+
+from etl.helpers import Names
+
+N = Names(__file__)
+
+
+def run(dest_dir: str) -> None:
+    dataset = catalog.Dataset.create_empty(dest_dir, N.garden_dataset.metadata)
+
+    table = N.garden_dataset["penn_world_table"]
+
+    # %%
+    # Specify sheet id and sheet (tab) name for a Google Sheet with the variables to keep
+    sheet_id = "1gbk8lBc4ZTjzE94pG8vgFX1Ta5baIQhpdD158GhPJsc"
+    sheet_name = "variable_metadata"
+
+    # Read in variable metadata as dataframe
+    url = f"https://docs.google.com/spreadsheets/d/{sheet_id}/gviz/tq?tqx=out:csv&sheet={sheet_name}"
```

Comment:
In ETL we try to avoid external sources to make it as replayable as possible and consistent in case someone changes something in the spreadsheet. Before we merge this we should copy the spreadsheet to the repo (as either csv or we could convert it to our YAML format).

Reply:
Ah, OK, let me see the alternative. Now it is only to filter the list of variables, so I can add something like a json file